src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -567,8 +572,10 @@ export default {
 	}
 }
 
-#carousel_exp >>> section > div:nth-child(2) {
-	display: none;
+@include breakpoint(xxlarge) {
```

Comment:
These are only hidden when the page is at full width and all loan cards are visible.